common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -5560,6 +5560,18 @@
     ],
     "sqlState" : "428EK"
   },
+  "THETA_INVALID_INPUT_SKETCH_BUFFER" : {
+    "message" : [
+      "Invalid call to <function>; only valid Theta sketch buffers are supported as inputs (such as those produced by the `theta_sketch_agg` function)."
+    ],
+    "sqlState" : "22546"
+  },
+  "THETA_INVALID_LG_NOM_ENTRIES" : {
+    "message" : [
+      "Invalid Theta sketch call; the `lgNomEntries` value must be between <min> and <max>, inclusive: <value>."
```

Comment:
Can we add "Invalid call to <function>" to this error message as well? It could help make the error message easier to understand, like the previous one.

Reply:
I agree with the idea, only concern is the function that throws this error is in the ThetaSketchUtils class [here](https://github.com/apache/spark/blob/e792ac2802ce8efbb772c33612429461d2f94ca5/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/ThetaSketchUtils.scala#L42). This function is unaware of the Theta sketch function calling it. Would we want the function name to be a parameter to the utils function?

Reply:
👍 let's pass the `prettyName` to the utils function(s) as needed if it's not too much extra work.

Reply:
yep this works.